### PR TITLE
Issue#204

### PR DESCRIPTION
**Pull request by blankgaming**
Here is the avatar name showing on the stage when the "show/hide button" is clicked
toggleshow

And here is the avatar name hidden on the stage when the "show/hide button" is clicked
togglehide

### CODE REVIEW

**Comment by JoeIceCream** [comment]
@blankgaming Am I allowed to review this? I helped create it.

File: app/assets/javascripts/cable.js [hunk 1]
```diff
@@ -25,6 +25,12 @@
     // Draw avatars
     App.state.avatars.forEach(function(avatar) {
       App.context.drawImage(avatar.image, avatar.x, avatar.y);
+        App.context.textAlign = "center";
+        App.context.textBaseline = "top";
+        console.log(avatar);
```
Comment (rohrlach) [inline app/assets/javascripts/cable.js]: Remove log messages

File: app/assets/javascripts/channels/avatars.coffee [hunk 2]
```diff
@@ -3,9 +3,15 @@ place = (data) ->
...
-      y: data['y'] - (img.width / 2)
+      x: data['x'] - (img.width / 2),
+      y: data['y'] - (img.height / 2),
+      name: data['name'],
+      text_x: data['x'],
+      text_y: data['y'] + (img.height / 2) + 5,
+      show: document.querySelector('#avatarName').value
     }
+    console.log(data)
+    console.log(img.width + ' ' + img.height)
```
Comment (rohrlach) [inline app/assets/javascripts/channels/avatars.coffee]: Here too

File: app/assets/javascripts/channels/avatars.coffee [hunk 3]
```diff
@@ -45,6 +55,24 @@ hold = (data) ->
       mirrorDiv.removeChild(mirrorDiv.childNodes[0])
     mirrorImg.src = data['file']
 
+name = (data) ->
+  nameBtn = document.querySelector '#avatarName'
+  if nameBtn.value == '1'
+    nameBtn.value = '0'
+  else
+    nameBtn.value = '1'
```
Comment (rohrlach) [inline app/assets/javascripts/channels/avatars.coffee]: Why are you doing this?  What is the use of setting the ```value``` attribute of a singular instance of ```#namBtn``` on every receive? What happens when the client receives a ```name``` action for an avatar they aren't holding?

**Comment by rohrlach** [comment]
@JoeIceCream @blankgaming If you both have worked on this PR - You could always request a review from @thongteav @charlie0605 or @alexanderglenn

**Comment by rohrlach** [comment]
@blankgaming what's going on with this?

**Comment by JoeIceCream** [comment]
Sorry for the lack of update Paul, I finished getting rid of the bugs but it took some time to work it out.

**Comment by rohrlach** [comment]
@JoeIceCream are you happy with the PR now?

**Comment by JoeIceCream** [comment]
@rohrlach Yes, everything seems fine to me now. We may want to add another issue that changes the font size to something more preferable.

**Comment by rohrlach** [comment]
@JoeIceCream feel free to create that as an issue.

Also, please resolve the conflicts listed

**Comment by JoeIceCream** [comment]
Hold on, that last commit is causing bugs.

**Pull request merged**